### PR TITLE
feat: Add otherRelevantPosition to collector profile

### PR DESCRIPTION
**Pull request by olerichter00**
Addresses [CX-2240]

🔴  Depends on https://github.com/artsy/gravity/pull/14812

## Description

Add otherRelevantPosition to collector profile.

[CX-2240]: https://artsyproduct.atlassian.net/browse/CX-2240?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by lordkiz — DISMISSED** [review]
LGTM

File: src/schema/v2/me/index.ts [hunk 1]
```diff
@@ -221,6 +221,9 @@ export const meType = new GraphQLObjectType<any, ResolverContext>({
     profession: {
       type: GraphQLString,
     },
+    otherRelevantPosition: {
+      type: GraphQLString,
```
Comment (lordkiz) [inline src/schema/v2/me/index.ts:225]: @olerichter00 you might want to add a description here.

Comment (olerichter00) [reply src/schema/v2/me/index.ts:225]: Good idea 👍

**Comment by dzucconi** [comment]
The schema wasn't updated here — plz make sure you don't no-verify as the git hooks to generate it won't run (this should probably be addressed some other way)

**Comment by olerichter00** [comment]
Thanks for the notification, @dzucconi 🙇 I updated the schema and created a new PR: https://github.com/artsy/metaphysics/pull/3675

**Pull request merged**